linode/manager
# fix: [M3-8787] - Fix Alignment for Backup Label in Add-ons Panel

## Description 📝
This PR aligns the **Backups** label with the checkbox component similar to the **Private IP** label and its checkbox component. It also makes the spacing between the label and the helper text uniform on both components.

## Changes  🔄
List any change relevant to the reviewer.
- The checkbox components are now top-aligned instead of being aligned vertically.
- Spacing is added between the **Private IP** label and helper text to match **Backups**

## Preview 📷
| Before  | After   |
| ------- | ------- |
**Desktop**
| ![desktop-before](https://github.com/user-attachments/assets/7b75d9ef-6a36-415a-8b8e-a1e380a9c9bf) | ![desktop-after](https://github.com/user-attachments/assets/3c612563-1367-4c92-8715-6af677706b87) |
| ![desktop-notice-before](https://github.com/user-attachments/assets/a61988db-8656-4251-9abf-810c91e83bcc) | ![desktop-notice-after](https://github.com/user-attachments/assets/2adbf93f-db94-4b56-93d0-59c70026cf39) |
**Mobile**
| ![mobile-before](https://github.com/user-attachments/assets/3eef4bc8-1215-4ce9-b65e-9eea5212a789) | ![mobile-after](https://github.com/user-attachments/assets/eca28378-59ae-4cac-8e16-df1c43d935d8) |
| ![mobile-notice-before](https://github.com/user-attachments/assets/1e2e9d31-0f0f-4045-a460-aa24bf93adb7) | ![mobile-notice-after](https://github.com/user-attachments/assets/e2a86d44-7cbf-4d11-8d00-caa686da264d) |

## How to test 🧪
### Reproduction steps

### Verification steps
(How to verify changes)
- Visit the Linode Create form on production and inspect the "Add ons" panel.
- Inspect the label alignment of the "Backups" and "Private IP" components.
- Launch this branch locally and In a separate window, inspect the same sections.

## As an Author I have considered 🤔
